lib/Controller/ChatController.php

```diff
@@ -1293,4 +1294,13 @@ protected function prepareResultArray(array $results, array $statuses): array {
 		}
 		return $output;
 	}
+
+	protected function createMentionString(string $type, string $id): string {
+		if ($type === Attendee::ACTOR_USERS || $type === Attendee::ACTOR_GUESTS) {
+			return $id;
+		}
+
+		// It's "group/admin" so we have to strip off the trailing "s" from the type
```

Comment:
```suggestion
		// It's "groups/admin" so we have to strip off the trailing "s" from the type
```

?? Otherwise I don't get the comment.